src/ray/core_worker/core_worker.h

```diff
@@ -211,17 +251,25 @@ class CoreWorker {
 
   /// Event loop where the IO events are handled. e.g. async GCS operations.
   boost::asio::io_service io_service_;
+
   /// Keeps the io_service_ alive.
   boost::asio::io_service::work io_work_;
+
   /// Timer used to periodically send heartbeat containing active object IDs to the
   /// raylet.
   boost::asio::steady_timer heartbeat_timer_;
 
   // Thread that runs a boost::asio service to process IO events.
   std::thread io_thread_;
 
+  // Task execution callback.
```

Comment:
Not necessary, but could we group together all of the members that have to do with task execution?

Reply:
Yes, will do that last once everything else is settled